wayland/install-wayland.txt

```diff
@@ -0,0 +1,72 @@
+NOTE: This article covers only Wayland - the protocol, which is still
+      experimental.
+      Bugs and quirks expected!
```

Comment:
```
NOTE: This article covers only Wayland - the protocol, which is still
      experimental. Bugs and quirks expected!
```

Reply:
Pages should start with the title/horizontal rule. This note should be moved somewhere below.